.github/workflows/Downstream.yml

```diff
@@ -37,6 +42,7 @@ jobs:
         with:
           version: ${{ matrix.julia-version }}
           arch: x64
+      - uses: julia-actions/cache@v1
```

Comment:
Why v1 instead of v3?

Reply:
Were you thinking of https://github.com/actions/cache? The latest version of https://github.com/julia-actions/cache is 1.5.3, so still major version v1. I know because I initially got confused by this too.

Reply:
Ah you're right. I thought this PR was just removing parameters to the action and didn't see the action itself changed.